app/_src/policies/meshmetric.md

```diff
@@ -231,6 +238,29 @@ scrape_configs:
         client_id: "prometheus-two"
 ```
 
+#### OpenTelemetry (experimental)
+
+```yaml
+backends:
+  - type: OpenTelemetry
+    openTelemetry: 
+      endpoint: http://otel-collector.observability.svc:4317
+```
+
+This configuration tells {{site.mesh_product_name}} Dataplane Proxy to push metrics to [OpenTelemetry collector](https://opentelemetry.io/docs/collector/).
+Dataplane Proxy will scrape metrics from Envoy and other [applications](/docs/{{ page.version }}/policies/meshmetric/#applications) 
+and push them to configured OpenTelemetry collector.
+
+When you configure application scraping make sure to specify `application.name` to utilize [OpenTelemetry scoping](https://opentelemetry.io/docs/concepts/instrumentation-scope/) 
+
+##### Limitations
+
+- You cannot configure scraping interval for OpenTelemetry. By default, it will publish metrics to collector every **30 seconds**. Ability to configure scraping interval in policy will be added in next release. 
```

Comment:
nit: in the future. I think we should avoid committing to specific releases in docs.

Reply:
changed